x/evm/types/types.go

```diff
@@ -51,6 +53,7 @@ func validateBurnerCode(burnerCode []byte) error {
 	case BurnerCodeHashV1:
 	case BurnerCodeHashV2:
 	case BurnerCodeHashV3:
+	case BurnerCodeHashV4:
```

Comment:
```suggestion
	case BurnerCodeHashV1, 
	        BurnerCodeHashV2, 
	        BurnerCodeHashV3, 
	        BurnerCodeHashV4:
	        break
```

Reply:
nothing wrong with the current implementation, I think this would just make it clearer that this is supposed to do nothing when it matches